src/plugins/auto/plugin.cpp

```diff
@@ -299,7 +299,8 @@ InferenceEngine::Parameter MultiDeviceInferencePlugin::GetMetric(const std::stri
                                                     RW_property(ov::auto_batch_timeout.name()),
                                                     RW_property(ov::hint::performance_mode.name()),
                                                     RW_property(ov::hint::num_requests.name()),
-                                                    RW_property(ov::intel_auto::device_bind_buffer.name())
+                                                    RW_property(ov::intel_auto::device_bind_buffer.name()),
+                                                    RW_property(ov::cache_dir.name())
```

Comment:
do we need the same for BATCH plugin?

Reply:
yes definitely, since I'm not sure about the impact on the existing batch related test cases, fix for batch will be in the next commit.

Reply:
ok, sounds good. Let's just not forget about it
@peterchen-intel